azurerm/resource_arm_healthcare_service_test.go

```diff
@@ -0,0 +1,130 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/terraform/helper/acctest"
+)
+
+func TestAccAzureRMHealthcareService_basic(t *testing.T) {
+	ri := acctest.RandIntRange(10000, 99999)
```

Comment:
Could we use the standard `tf.AccRandTimeInt` here? if need /10 or /100 for it to fit

Reply:
Done

Reply:
For this and the data source test, we switched to `tf. AccRandTimeInt / 10` and changed the prefix to testacc for a format of `testacc19102406571368976`, as suggested. Hopefulyy this will be random enough to be unique and still be 24 chars.